### ISSUE TITLE
Can't create init user with examples-next/todo using prisma adapter: trackingFields's updatedBy field problem

### ISSUE THREAD

**Issue by MurzNN** [keystonejs/keystone#5014]
# Bug report

## Describe the bug

When trying to test "examples-next/todo" demo with prisma_postgresql adapter - I got the error on `/init` page when try to create the first user:
```
Invalid `prisma.user.create()` invocation: Query interpretation error. Error for binding '1': AssertionError("[Query Graph] Expected a valid parent ID to be present for a nested connect on a one-to-many relation.")
```

Seems the source of problem is that in `packages/adapter-prisma/lib/adapter-prisma.js` function `_create()` handles wrong the `null` values in relationship fields.
For the input data like this:
```
createdAt:Thu Mar 04 2021 16:21:12 GMT+0300 (Moscow Standard Time)
email:'korepov@example.com'
name:'korepov'
password:'$2a$10$nbCLgNg/GSVLTqlzOwiTyuJoodUPiwOqZQAZxWf.S5ifsKvjBftAu'
updatedAt:Thu Mar 04 2021 16:21:12 GMT+0300 (Moscow Standard Time)
updatedBy:null
```
it generates the model with this problematic part:
```
updatedBy: {
  connect: {
    id: 0
  }
}
```
instead of excluding the relation field with null value, or at least `id: null`.

## To Reproduce

Steps to reproduce the behavior. Please provide code snippets or a repository:

1. Launch the "examples-next/todo" demo with `adapter: 'prisma_postgresql'`
2. Open the `/init` page, fill the form
3. Submit the form.
4. See the error:
```
Invalid `prisma.user.create()` invocation: Query interpretation error. Error for binding '1': AssertionError("[Query Graph] Expected a valid parent ID to be present for a nested connect on a one-to-many relation.")
```

## Expected behavior

Initial user should be created.

**Comment by MurzNN** [comment]
Simple hack like this:
```js
  async _create(_data) {
    if (_data.updatedBy == null) {
      delete (_data.updatedBy);
    }
    let model = {
      data: mapKeys(_data, (value, path) =>
        this.fieldAdaptersByPath[path] && this.fieldAdaptersByPath[path].isRelationship
          ? {
              connect: Array.isArray(value)
                ? value.map(x => ({ id: Number(x) }))
                : { id: Number(value) },
            }
          : this.fieldAdaptersByPath[path] && this.fieldAdaptersByPath[path].gqlToPrisma
          ? this.fieldAdaptersByPath[path].gqlToPrisma(value)
          : value
      ),
      include: this._include(),
    };
    return this.model.create(model);
  }
```
solves the initial user create problem.

**Comment by MurzNN** [comment]
Here is my PR https://github.com/keystonejs/keystone/pull/5015 that try to fix this via filling empty object `{}` as value, seems it fixes this issue. Can you please lookup it?

**Comment by timleslie** [comment]
Closed by #5016
